README.md

```diff
@@ -185,45 +194,313 @@ interface LookerEmbedUser {
   first_name?: string
   last_name?: string
   session_length: number
-  force_logout_login?: boolean,
+  force_logout_login?: boolean
   permissions: LookerUserPermission[]
   models: string[]
   group_ids?: number[]
   external_group_id?: string
-  user_attributes?: {[key: string]: any}
-  access_filters: {[key: string]: any}
+  user_attributes?: { [key: string]: any }
+  access_filters: { [key: string]: any }
 }
 ```
 
+<a name='cookieless' id='cookieless'></a>
+
+## Cookieless Embed
+
+Note that cookieless embed does not yet support the use of `withUrl`. An attempt to do use this functionality will result in an error being thrown.
+
+Looker cookieless embed allows the Looker application to be embedded by an html page that is served from a different domain than the Looker host. With SSO embed, to avoid third party cookie blocking, the Looker application must be served from a sub domain of the hosting application OR the user must enable third party cookies in the browser. Enabling cookieless embedding is documented in more detail [here](https://cloud.google.com/looker/docs/r/sdk/cookieless-embed). Cookieless embed is available with Looker version 22.18 and above.
```

Comment:
```suggestion
Looker cookieless embed allows the Looker application to be embedded by an html page that is served from a different domain than the Looker host. With SSO embed, to avoid third party cookie blocking, the Looker application must be served from a sub domain of the hosting application OR the user must enable third party cookies in the browser. Enabling cookieless embedding is documented in more detail [here](https://cloud.google.com/looker/docs/r/sdk/cookieless-embed). Cookieless embed is available with Looker version 22.20 and above.
```